settings/settings.pro

```diff
@@ -1,22 +1,21 @@
 TEMPLATE = aux
 
 entries.path = /usr/share/jolla-settings/entries
-entries.files = onyx-camera-settings.json
+entries.files = camera-settings.json
 
-pages.path = /usr/share/jolla-settings/pages/onyx-camera-settings
-pages.files = settings.qml camera-resolutions.json
+pages.path = /usr/share/jolla-settings/pages/camera-settings-plugin

```

Comment:
extra -plugin here.
will fail as entry points to "path": "system_settings/system/camera-settings",